wordpress-mobile/WordPress-iOS
# Properly sizes gallery images

Related: #1260 

This PR causes images within a gallery to have their `src` attributes modified with parameters to resize them more appropriately for the current device screen. This solves an issue with images in mosaic galleries being of different sizes, mostly on private sites only.

![img_0522](https://cloud.githubusercontent.com/assets/373903/17378897/7ece7c4c-5985-11e6-91f2-f35f153761fa.PNG)

("Before" screenshot was taken on an older branch so the header styling does not match)

### To test:

Ensure images load properly for the following scenarios in Reader:
1. A post with no gallery but several images. (images should remain unresized)
2. A post on a public site with a gallery of 1 or more images. (images will be resized and pulled from Photon)
3. A post on a private site with a gallery of 1 or more images. (images will be resized).


Needs review: @aerych
